app_dart/lib/src/request_handlers/check_flaky_tests_and_update_github.dart

```diff
@@ -0,0 +1,394 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
sorry forgot to mention earlier, could we move corresponding logics to different services, like /lib/src/service/github_service.dart, or the bigquery.dart? The structure should be more clear and easier to follow that way. And it would be easier to add corresponding tests in separate locations.

Reply:
sounds good